internal/pkg/agent/application/upgrade/step_unpack.go

```diff
@@ -200,6 +199,7 @@ func untar(log *logger.Logger, version string, archivePath string) (string, erro
 				return "", errors.New(err, "TarInstaller: creating file "+abs, errors.TypeFilesystem, errors.M(errors.MetaKeyPath, abs))
 			}
 
+			//nolint:gosec // legacy
```

Comment:
for future: would be better to narrow down a rule for gosec, let's keep it here as is for now